documents/motions/10.2022.3 - Disciplinary Committee.md

```diff
@@ -1,34 +1,34 @@
 # SUBMISSION OF PROPOSED MOTION
 
 **Motion number:** 10.2022.3  
-**Subject:** WCA Disciplinary Committee  
-**Intent:** Rights and duties of the WCA Disciplinary Committee  
+**Subject:** WCA Integrity Committee  
+**Intent:** Rights and duties of the WCA Integrity Committee  
 **Submitted by:** WCA Board  
-**Date:** May 1, 2022  
+**Date:** TBC, 2024   
 
 # Motion
 
-The WCA Disciplinary Committee is an Advisory Committee of the WCA.
+The WCA Integrity Committee is an Advisory Committee of the WCA.
 
 1. The WCA Disciplinary Committee has the following rights and duties:
    1. Investigations
       1. Performing independent investigations regarding WCA Staff and Registered Speedcubers.
-         1. The subjects of the investigations are incidents during WCA Competitions, incidents related to WCA Competitions, or incidents within the WCA Community. Incidents are alleged violations of the Spirit and/or Regulations of the WCA.
+         1. The subjects of the investigations are incidents during WCA Competitions, incidents related to WCA Competitions, or incidents within the WCA Community. Incidents are alleged violations of the Spirit, Code of Conduct, Code of Ethics and/or Regulations of the WCA.
+         2. The investigations can be initiated by the WCA Integrity Committee itself, by the WCA Board.
       2. Delivering an independent report on the investigations.
-      3. Involving the WCA Ethics Committee in case any WCA Staff Member is suspected of acts against the Spirit of the WCA or not compliant with the Code of Ethics.
-      4. Making decisions on disciplinary actions in relation to the investigated incidents. See [WCA Motion : Suspensions and other Sanctions](wcadoc{documents/motions/15.2022.1 - Suspensions and other Sanctions.pdf}).
+      3. Making decisions on disciplinary actions in relation to the investigated incidents. See [WCA Motion : Suspensions and other Sanctions](wcadoc{documents/motions/15.2022.1 - Suspensions and other Sanctions.pdf}).
    2. Confidant
       1. Being the confidential contact for Registered Speedcubers and WCA Staff, in case of important personal matters that are related to WCA activities.
    3. Information and administration
       1. Informing the WCA Board on investigations, decisions, and disciplinary actions.
-      2. Publishing public information on the investigations, decisions, and disciplinary actions.
+      2. Publishing public information on the investigations, decisions, and disciplinary actions where necessary.
```

Comment:
Assume we are being deliberate in who decides what necessary is? I.e. I assume it would be a decision of the committee/Board?

Reply:
Yeah, the reason to add where necessary is a carry over of what is currently within the WEC motion so that they don't need to publish reports on everything just the relevant stuff. Keeping it vague here allows that decision to be made by the committee or the Board